ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -385,6 +385,7 @@ private void nUD_PeakCountMinThreshold_ValueChanged(object sender, EventArgs e)
                 Parallel.ForEach(SaveState.lollipop.et_peaks, p =>
                 {
                     p.Accepted = p.peak_relation_group_count >= SaveState.lollipop.min_peak_count_et;
+                    p.count_nearby_decoys(SaveState.lollipop.ed_relations.Values.SelectMany(v => v).ToList());
```

Comment:
This method sets the decoy relations in the range of the peak to be accepted/not accepted based on if the peak is accepted/not accepted, so if peak acceptance changes the decoy relations should be reset